### DISCUSSION TITLE
what mean Key_Shared about "STICKY"?

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#19522] [Q&A] [answered]
where STICKY??

**Comment by startjava** [comment]
i see only can custom consumer self hash range，but i no see where embody about STICKY

**Comment by tisonkun** [comment, marked as answer]
In Shared mode, messages can be randomly consumed by any consumer in the subscription.

But in Key_Shared mode, messages with the same key will always be consumed by the same consumer.

This is what "STICKY" means.
